themes/bootstrap3/templates/RecordDriver/DefaultRecord/cover.phtml

```diff
@@ -4,20 +4,27 @@
   if ($linkAttributes && !isset($linkAttributes['class'])) {
     $linkAttributes['class'] = 'record-cover-link';
   }
-  $alt = $linkAttributes ? trim($this->driver->tryMethod('getTitle') ?? '') : '';
+  if ($linkAttributes && !$this->linkUnique) {
+    $linkAttributes['tabindex'] = '-1';
```

Comment:
Since this PR was originally created, the behavior of the cover images in the search results has changed; instead of linking to the record page, it now pops up a larger version of the image just like on the record page. Now that the behavior is consistent everywhere, do we no longer need this special case?

Reply:
In that case I don't think we need this special case. There might even be many more changes we don't need anymore.

When I originally opened this pull request there was 0af81e1ba02f09ee0b4122581af20dec236d283f to fix redundant links and 1a93ec341c8de803912cf4e719637a106aea286b to fix redundant alternative texts. If the links are no longer redundant, the only thing we really need to change are the alt texts.

Reply:
Sounds like that could greatly simplify everything. I think if we can eliminate the linkUnique flag that will make the changes here (and the code in general) a bit less potentially confusing.

I don't mind taking a stab at removing $linkUnique if that would be helpful. Is the idea then that we'll leave the aria-hidden div in place for non-linked cover images since those are not useful/meaningful, but we'll leave linked cover images unhidden with the normal tab order, but with appropriate alt text?

Reply:
I recommend doing this from scratch, just changing the alt texts.

Reply:
Okay, I have rebased and simplified the code -- what do you think of this version?

Reply:
I must admit my head is spinning from all the revisions this have been going through. The code looks reasonable to me, but the devil is in the details.

Reply:
Agreed, it has been a long and twisting road to get this far. But I think the bottom line is that we're using uniform alt text on the cover images now because they're no longer being used as record links, and we're aria-hiding the thumbnails when they don't serve as links since they're purely decorative in that instance... and those seem like reasonable changes that should be safe to commit. If there are additional adjustments needed, we can always follow up later!